test/LaunchDarkly.EventSource.Tests/EventSourceTests.cs

```diff
@@ -16,64 +16,15 @@ public class EventSourceTests
         private readonly Uri _uri = new Uri("http://test.com");
         private readonly TimeSpan _defaultReadTimeout = TimeSpan.FromSeconds(1);
 
-        [Fact]
-        public void Exponential_backoff_should_not_exceed_maximum()
```

Comment:
These three tests are now in a separate file, because they don't test `EventSource`, they just test the backoff helper class.